data.js

```diff
@@ -1,29 +1,42 @@
 export const data = {
-     first : {
-        photo : 'https://assets.entrepreneur.com/content/3x2/2000/20181221212044-GettyImages-970432726.jpeg?width=300&crop=1:1',
-        name : 'Queen Zulu',
-        filds :'Professor in computer science',
-        texte : 'Thank you very much for this platform because it pushes our students to work hard and face educational difficulties while improving their knowledge.'
-    },
-    second:{
-        photo : 'https://presspage-production-content.s3.amazonaws.com/uploads/2262/500_monroy845x.jpg?x=1606759573293',
-        name : 'Mr. Professor',
-        filds :'Professor in computer science',
-        texte : 'Thank you very much for this platform because it pushes our students to work hard and face educational difficulties while improving their knowledge.'
-    }
-    ,
-    third:{
-        photo : 'img/daniel.jpg',
-        name : 'Safari Hamuli',
-        filds :'Professor at Free Courses',
-        texte : 'Since I learned programming, this platform is the only one where learners learn and teach at the same time.'
-    }
-    ,
-   fourth:{
-    photo : 'https://assets.entrepreneur.com/content/3x2/2000/20181221212044-GettyImages-970432726.jpeg?width=300&crop=1:1',
-    name : 'Queen Zulu',
-    filds :'Professor in computer science',
-    texte : 'Thank you very much for this platform because it pushes our students to work hard and face educational difficulties while improving their knowledge.'
+  first: {
+    photo: 'https://assets.entrepreneur.com/content/3x2/2000/20181221212044-GettyImages-970432726.jpeg?width=300&crop=1:1',
+    name: 'Queen Zulu',
+    filds: 'Professor in computer science',
+    texte: 'Thank you very much for this platform because it pushes our students to work hard and face educational difficulties while improving their knowledge.',
+  },
+  second: {
+    photo: 'https://presspage-production-content.s3.amazonaws.com/uploads/2262/500_monroy845x.jpg?x=1606759573293',
+    name: 'Mr. Professor',
+    filds: 'Professor in computer science',
+    texte: 'Thank you very much for this platform because it pushes our students to work hard and face educational difficulties while improving their knowledge.',
+  },
+  third: {
+    photo: 'img/daniel.jpg',
+    name: 'Safari Hamuli',
+    filds: 'Professor at Free Courses',
+    texte: 'Since I learned programming, this platform is the only one where learners learn and teach at the same time.',
+  },
+  fourth: {
+    photo: 'https://assets.entrepreneur.com/content/3x2/2000/20181221212044-GettyImages-970432726.jpeg?width=300&crop=1:1',
+    name: 'Queen Zulu',
+    filds: 'Professor in computer science',
+    texte: 'Thank you very much for this platform because it pushes our students to work hard and face educational difficulties while improving their knowledge.',
```

Comment:
- [ ] Everything is good you are almost there but you need to change small things. Please implement it.
- [ ] In the featured speaker It would be great if you increase the size of every card and please implement the small background that is on the photo section. Please see the image below for reference. As the previous reviewer did not mention every section so I am telling you in detail. Please implement it.
### Your design
![your](https://user-images.githubusercontent.com/81727988/205037750-1c2dc8d9-b22a-4539-ab92-cf006d9979e4.png)
###  Figma design
![figma](https://user-images.githubusercontent.com/81727988/205037790-412fb663-b1b7-48e3-8a1c-317ad81740ec.png)